modules/ROOT/pages/promote-api-task.adoc

```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Before promoting an API instance to another environment, ensure that you have the `Manage APIs Configuration` permission.
+Before promoting an API instance to another environment, ensure that you have the `Manage APIs Configuration` permission. 
 
-You can only promote an entity, such as policies or alerts, if you have both the permissions to view the entity and to manage its destination. For example, you must have the `View Policies` and `Manage Policies` permissions to promote a policy. 
+To promote an entity, you must have permissions to view and manage the entity. For example, you must have the `View Policies` and `Manage Policies` permissions to promote a policy.
 
 To promote an API instance:
```

Comment:
I think we should remove this right? I can't remember, but I don't think it leads into anything.